helpers/GUIDELINES.md

```
@@ -139,6 +140,50 @@ const data = await options.queryAllium(`SELECT ...`);
 import { queryIndexer } from '../helpers/indexer';
 ```
 
+### Adding Generic EVM Chain Metrics
+
+Use `helpers/evmChainFees.ts` when an EVM chain needs chain-level fees, revenue, transaction count, gas used, and active users from RPC receipts.
+
+1. Add the chain config to `EVM_CHAIN_METRIC_CONFIGS`:
+```typescript
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)